unified-runtime/scripts/core/device.yml

```diff
@@ -417,6 +417,8 @@ etors:
             available for this device.
     - name: KERNEL_SET_SPECIALIZATION_CONSTANTS
       desc: "[$x_bool_t] support the $xKernelSetSpecializationConstants entry point"
+    - name: BFLOAT16_CONVERSIONS_INTEL
+      desc: "[$x_bool_t] support for native bfloat16 conversions"
```

Comment:
To avoid rewriting all of the enum values, move this new entry to the end of the array, i.e. after line 451.

Reply:
Could you move this to the bottom and run the generate again please?